physlr/physlr.py

```diff
@@ -1517,6 +1517,45 @@ def determine_molecules_partition_split_merge(g, u):
                     bi_connected_component, strategy=0)
                 ]
 
+    @staticmethod
+    def determine_molecules_extensive(g, u):
+        """
+        Assign the neighbours of this vertex to molecules
+        by Applying a queue of different algorithms on top of each other.
+        """
+        from collections import deque
+
+        communities = [g[u].keys()]
+        communities_final = []
+
+        alg_list = deque(["bc", "k3", "bc", "cos", "bc", "sqCos", "bc"])
+        while alg_list:
```

Comment:
```suggestion
        for algorithm in ["bc", "k3", "bc", "cos", "bc", "sqCos", "bc"]:
```

Reply:
Have you tested does having the `bc` between each algorithm have an effect? Does it give better separation than say `["bc", "k3", "cos", "sqCos"]`?

Reply:
Nope :D I'll test but Inanc suggested that since bc is extremely faster than the others it wouldn't be a pain to do so.

Reply:
I have restored the code to `["bc", "k3", "cos", "sqCos"]` for now, and I will test idea of multiple bcs later.